docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -1418,34 +1418,80 @@ Return low-level information on the image `name`
     Content-Type: application/json
 
     {
-         "Created": "2013-03-23T22:24:18.818426-07:00",
-         "Container": "3d67245a8d72ecf13f33dffac9f79dcdf70f75acb84d308770391510e0c23ad0",
-         "ContainerConfig":
-                 {
-                         "Hostname": "",
-                         "User": "",
-                         "AttachStdin": false,
-                         "AttachStdout": false,
-                         "AttachStderr": false,
-                         "Tty": true,
-                         "OpenStdin": true,
-                         "StdinOnce": false,
-                         "Env": null,
-                         "Cmd": ["/bin/bash"],
-                         "Dns": null,
-                         "Image": "ubuntu",
-                         "Labels": {
-                             "com.example.vendor": "Acme",
-                             "com.example.license": "GPL",
-                             "com.example.version": "1.0"
-                         },
-                         "Volumes": null,
-                         "VolumesFrom": "",
-                         "WorkingDir": ""
-                 },
-         "Id": "b750fe79269d2ec9a3c593ef05b4332b1d1a02a62b4accb2c21d589ff2f5f2dc",
-         "Parent": "27cf784147099545",
-         "Size": 6824592
+      "GraphDriver" : {
+        "Name" : "aufs",
+          "Data" : null
+      },
+      "ContainerConfig" : {
+        "NetworkDisabled" : false,
+        "Hostname" : "e611e15f9c9d",
+        "AttachStdout" : false,
+        "AttachStdin" : false,
+        "PublishService" : "",
+        "AttachStderr" : false,
+        "Entrypoint" : null,
+        "Tty" : false,
+        "Volumes" : null,
+        "Cmd" : [
+          "/bin/sh",
+          "-c",
+          "#(nop) CMD [\"/bin/bash\"]"
+        ],
+        "Image" : "d74508fb6632491cea586a1fd7d748dfc5274cd6fdfedee309ecdcbc2bf5cb82",
+        "MacAddress" : "",
+        "ExposedPorts" : null,
+        "StdinOnce" : false,
+        "User" : "",
+        "Domainname" : "",
+        "Env" : null,
+        "WorkingDir" : "",
+        "VolumeDriver" : "",
+        "OnBuild" : null,
+        "Labels" : {},
```

Comment:
Looks like the labels got lost in the example output; could you make sure the example output contains some labels?

See my earlier comment for some instructions to create images with the labels that were present previously; https://github.com/docker/docker/pull/13185#discussion_r32813857